api/hearings/models/refData.model.ts

```diff
@@ -6,6 +6,7 @@ export interface RefDataModel {
   hintTextCY: string;
   order: number;
   parentKey: string;
+  child_nodes?: RefDataModel[];
```

Comment:
Does the data from the api come back as `snake_case`? If not can we stick to `camelCase`?

Reply:
I think we need a quick chat about coding standards @rayliangatos and @kuda1992

Reply:
yes this child_nodes is from api,

Reply:
@rayliangatos @waynepHMCTS . If that's the case I will resolve the conversation.